.gitignore

```diff
@@ -21,3 +21,6 @@ erl_crash.dump
 
 # Redis db
 dump.rdb
+
+# The directory used by Elixir Language Server
+.elixir_ls
```

Comment:
Can remove if needed

Reply:
Let's keep it, it seems sensible to have it here.